docker/transformers-all-latest-gpu/Dockerfile

```diff
@@ -50,7 +50,22 @@ RUN python3 -m pip install --no-cache-dir git+https://github.com/huggingface/acc
 
 RUN python3 -m pip install --no-cache-dir git+https://github.com/huggingface/peft@main#egg=peft
 
-# For bettertransformer
+# Add bitsandbytes for mixed int8 testing
```

Comment:
Can you move that to the new quantization dockerfile?

Reply:
done